src/test/java/reiff/gameoflife/GameOfLifeTest.java

```diff
@@ -0,0 +1,48 @@
+package reiff.gameoflife;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class GameOfLifeTest {
+        @Test
+        public void string() {
+            //given
+            GameOfLife game = new GameOfLife(3, 3);
+
+            //when
+            String actual = game.toString();
+
+            //then
+            assertEquals("000\n000\n000\n", actual);
+        }
+
+        @Test
+    public void setCell() {
+            //given
+            GameOfLife game = new GameOfLife(3, 3);
+
+            //when
+            game.setCell(1, 2, 1);
+
+            //then
+            assertEquals("000\n000\n010\n", game.toString());
+        }
+
+
+        @Test
+    public void nextGen() {
+        //given
+        GameOfLife game = new GameOfLife(3, 3);
+        game.setCell(1, 1, 1);
+        game.setCell(2, 1, 1);
+        game.setCell(0, 0, 1);
```

Comment:
3 horizontal or 3 vertical cells

Reply:
I should do either "000\n111\n000\n" or "010\n010\n010\n"?

Reply:
Yes

Reply:
Thank you